### ISSUE TITLE
Uncaught 400 Bad Request / wrong parameter use

### ISSUE THREAD

**Issue by innersix** [PAhelper/PhoenixAdult.bundle#1172]
Scene URL: https://www.21sextury.com/en/video/gapeland/Anal-training-of-Karen--Ioana/97298
Filename: `Gapeland - 2011-05-12 - 97298 - Karen Ioana - Anal training of Karen & Ioana.wmv`
Log excerpt:
```
2021-01-22 15:55:38,447 (-4fa01ba0) :  INFO (agentkit:961) - Searching for matches for {'openSubtitlesHash': '7efac9330e14d541', 'name': 'Gapeland 2011 05 12 97298 Karen Ioana Anal Training of Karen & Ioana', 'filename': '%2Fmedia%2Fwdpassport1tb%2FAdult%2FVideos%2FGapeland%20-%202011-05-12%20-%2097298%20-%20Karen%20Ioana%20-%20Anal%20training%20of%20Karen%20%26%20Ioana%2Ewmv', 'plexHash': 'ca280bfa3a8ebee28b0f6511feda4a4fbbae0c55', 'duration': '-1', 'id': '564'}
2021-01-22 15:55:38,448 (-4fa01ba0) :  DEBUG (networking:143) - Requesting 'http://127.0.0.1:32400/library/metadata/564/tree'
2021-01-22 15:55:38,459 (-4fa01ba0) :  INFO (__init__:58) - ***MEDIA TITLE [from media.name]*** Gapeland 2011 05 12 97298 Karen Ioana Anal Training of Karen & Ioana
2021-01-22 15:55:38,460 (-4fa01ba0) :  INFO (PAsearchSites:52) - mediaTitle w/ possible abbreviation: Gapeland 2011 05 12 97298 Karen Ioana Anal Training of Karen & Ioana
2021-01-22 15:55:38,498 (-4fa01ba0) :  INFO (PAsearchSites:60) - mediaTitle w/ possible abbrieviation fixed: Gapeland 2011 05 12 97298 Karen Ioana Anal Training of Karen & Ioana
2021-01-22 15:55:38,526 (-50a03ba0) :  INFO (PAutils:203) - debug_dir is "/var/lib/plexmediaserver/Library/Application Support/Plex Media Server/Plug-in Support/Data/com.plexapp.agents.phoenixadult/debug_data/22-01-2021"
2021-01-22 15:55:38,531 (-4fa01ba0) :  INFO (PAsearchSites:45) - Site found
2021-01-22 15:55:38,532 (-4fa01ba0) :  INFO (PAsearchSites:72) - ^^^^^^^ siteNum: 370
2021-01-22 15:55:38,533 (-4fa01ba0) :  INFO (PAsearchSites:73) - ^^^^^^^ Shortening Title
2021-01-22 15:55:38,533 (-50a03ba0) :  INFO (PAutils:212) - GZip request saved as "12a690788129406883b354281a7dbf1f.gz"
2021-01-22 15:55:38,538 (-4fa01ba0) :  INFO (PAsearchSites:96) - Search Title (before date processing): 2011 05 12 97298 Karen Ioana Anal Training of Karen & Ioana
2021-01-22 15:55:38,540 (-50a03ba0) :  INFO (network1service:166) - Artwork found: 6
2021-01-22 15:55:38,541 (-4fa01ba0) :  INFO (__init__:60) - {'siteName': 'gapeland', 'searchDate': '2011-05-12', 'siteNum': 370, 'searchTitle': '97298 Karen Ioana Anal Training of Karen & Ioana'}
2021-01-22 15:55:38,543 (-50a03ba0) :  INFO (PAsearchSites:132) - Found https://media-public-ht.project1content.com/m=ea_aGJcWx/d2f/03e/a58/ec0/4b1/e8d/197/61d/81e/ad5/37/poster/poster_01.jpg in posters collection
…
2021-01-22 15:55:39,279 (-4fa01ba0) :  INFO (PAutils:107) - Requesting POST "https://tsmkfa364q-dsn.algolia.net/1/indexes/*/queries?x-algolia-application-id=TSMKFA364Q&x-algolia-api-key=MTdmMTU3MDI2MDhiODgwMzVjMWZhOGYxN2I1MWJiZjdmODY4OGY4N2FiZTMzZWJjMGM1YWE1MWI3NmQxZDcyNHZhbGlkVW50aWw9MTYxMTMzNDUzOSZyZXN0cmljdEluZGljZXM9YWxsJTJBJmZpbHRlcnM9c2VnbWVudCUzQWFkdWx0dGltZQ=="
2021-01-22 15:55:39,419 (-4fa01ba0) :  INFO (PAutils:203) - debug_dir is "/var/lib/plexmediaserver/Library/Application Support/Plex Media Server/Plug-in Support/Data/com.plexapp.agents.phoenixadult/debug_data/22-01-2021"
2021-01-22 15:55:39,421 (-4fa01ba0) :  INFO (PAutils:212) - GZip request saved as "c794d735096943658ebc96c0dbb61680.gz"
2021-01-22 15:55:39,422 (-4fa01ba0) :  CRITICAL (agentkit:1018) - Exception in the search function of agent named 'PhoenixAdult', called with keyword arguments {'openSubtitlesHash': '7efac9330e14d541', 'name': 'Gapeland 2011 05 12 97298 Karen Ioana Anal Training of Karen & Ioana', 'filename': '%2Fmedia%2Fwdpassport1tb%2FAdult%2FVideos%2FGapeland%20-%202011-05-12%20-%2097298%20-%20Karen%20Ioana%20-%20Anal%20training%20of%20Karen%20%26%20Ioana%2Ewmv', 'plexHash': 'ca280bfa3a8ebee28b0f6511feda4a4fbbae0c55', 'duration': '-1', 'id': '564'} (most recent call last):
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-3c3adfcb4/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/api/agentkit.py", line 1011, in _search
    agent.search(*f_args, **f_kwargs)
  File "/var/lib/plexmediaserver/Library/Application Support/Plex Media Server/Plug-ins/PhoenixAdult.bundle/Contents/Code/__init__.py", line 88, in search
    provider.search(results, lang, siteNum, search)
  File "/var/lib/plexmediaserver/Library/Application Support/Plex Media Server/Plug-ins/PhoenixAdult.bundle/Contents/Code/networkGammaEntOther.py", line 58, in search
    score = 100 - Util.LevenshteinDistance(searchData.date, releaseDate)
  File "/var/lib/plexmediaserver/Library/Application Support/Plex Media Server/Plug-ins/PhoenixAdult.bundle/Contents/Code/networkGammaEntOther.py", line 21, in getAlgolia
    return data['results'][0]['hits']
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-3c3adfcb4/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/code/sandbox.py", line 108, in <lambda>
    _getitem_           = lambda x, y: x.__getitem__(y),
KeyError: 'results'

```

HTTP Request attached: [c794d735096943658ebc96c0dbb61680.gz](https://github.com/PAhelper/PhoenixAdult.bundle/files/5857068/c794d735096943658ebc96c0dbb61680.gz)

Gist: request payload
```
{"requests": [{"params": "query=Karen Ioana Anal Training of Karen & Ioana", "indexName": "all_scenes"}]}
```

Response:
```
> HTTP/1.1 400 Bad Request

{"message":"Unknown parameter: Ioana","status":400}
```

**Comment by DirtyRacer1337** [comment]
`&` those symbol gives error

**Comment by DirtyRacer1337** [comment]
Fixed in f9b9688d3fdaf21e4e4b72b0c2d4d6c4871ef176
